spec/config_manager/http_project_config_manager_spec.rb

```diff
@@ -468,4 +463,56 @@
       expect(@http_project_config_manager.optimizely_config['revision']).to eq('81')
     end
   end
+
+  describe 'datafile authentication' do
+    it 'should add authorization header when auth token is provided' do
+      allow(Optimizely::Helpers::HttpUtils).to receive(:make_request)
+      @http_project_config_manager = Optimizely::HTTPProjectConfigManager.new(
+        sdk_key: 'valid_sdk_key',
+        datafile_access_token: 'the-token'
+      )
+      sleep 0.1
+      expect(Optimizely::Helpers::HttpUtils).to have_received(:make_request).with(anything, anything, anything, hash_including('Authorization' => 'Bearer the-token'), anything)
+    end
+
+    it 'should add authorization header when auth token is provided' do
+      allow(Optimizely::Helpers::HttpUtils).to receive(:make_request)
+      @http_project_config_manager = Optimizely::HTTPProjectConfigManager.new(
+        sdk_key: 'valid_sdk_key',
+        datafile_access_token: 'the-token'
+      )
+      sleep 0.1
+      expect(Optimizely::Helpers::HttpUtils).to have_received(:make_request).with(anything, anything, anything, hash_including('Authorization' => 'Bearer the-token'), anything)
+    end
```

Comment:
Just out of curiosity, is there a reason why this part of the spec is duplicated? Seems like the two blocks are exactly the same. 
Wondering if this is instead supposed to test the case where an invalid SDK key is provided (not sure if that is handled / error-checked elsewhere)

Reply:
Hey @pthompson127 ... good eye... its just a plain stupid mistake :). i wil fix it.

Reply:
@zashraf1985 ahh gotcha :)